proto/data.go

```diff
@@ -316,6 +317,69 @@ func (v *Value) SetProto(msg gogoproto.Message) error {
 	return nil
 }
 
+// SetTime encodes the specified time value into the bytes field of the
+// receiver and sets the tag.
+func (v *Value) SetTime(t time.Time) error {
+	var err error
+	if v.Bytes, err = t.MarshalBinary(); err != nil {
```

Comment:
I don't think the offset we store matters much here (as long as we store a fixed offset and not a zone name), but I would prefer to only store timestamps in UTC. We'll have to do conversions on load in any case (since the session time zone when we read may not be the same as when we wrote), and fixing the offset at 0 for the stored form seems better than letting it vary.